README.md

```diff
@@ -1,3 +1,87 @@
-# propguard
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
 
-Currently only two software members really know how to start the drone and enable its autonomous features. We need to change that. Propguard is a webapp that teaches people how to use Dronekit and Python to start and launch the drones.
+<!-- PROJECT LOGO AND TITILE -->
+<br />
+<p align="center">
+    <h2 align="center">Propguard</h2>
+    <p align="center">Currently only two software members really know how to start the drone and enable its autonomous features. We need to change that. Propguard is a webapp that teaches people how to use Dronekit and Python to start and launch the drones.</p>
```

Comment:
On reevaluation, lets ditch the "two software members" and just keep a version of the last sentence. It's too specific and won't be true all the time.